src/server/scripts/Northrend/Naxxramas/instance_naxxramas.cpp

```diff
@@ -25,6 +25,8 @@
 #include "ScriptedCreature.h"
 #include "naxxramas.h"
 
+static inline constexpr uint8_t HorsemanCount = 4;
```

Comment:
uint8, also constexpr is implicitly inline in c++ so you do not need to specify it here.